soccer/gameplay/tactics/coordinated_pass.py

```diff
@@ -86,7 +92,7 @@ def receive_point(self, value):
 
 
     def on_enter_running(self):
-        receiver = skills.pass_receive.PassReceive()
+        receiver = self.skillreceiver
```

Comment:
I think we need to `restart()` the skillreceiver here - if the CoordinatedPass is restarted as of now, it'll use the same instance of skillreceiver, which will be in some sort of completed state.